Tests/SwiftSyntaxBuilderTest/ExpressibleBuildablesTests.swift

```diff
@@ -0,0 +1,95 @@
+final class ExpressibleBuildablesTests: XCTestCase {
+    func testExpressibleAsMemberDeclListItem() {
+        let myStruct = StructDecl(identifier: "MyStruct", members: MemberDeclBlock(membersBuilder: {
+            VariableDecl(.var, name: "myFirstVar", type: "Int")
+
+            VariableDecl(letOrVarKeyword: .let, bindingsBuilder: {
+                PatternBinding(pattern: IdentifierPattern(identifier: .identifier("mySecondLet")),
+                               typeAnnotation: TypeAnnotation(type: SimpleTypeIdentifier("CGFloat")))
+            })
+
+            MemberDeclListItem(decl: VariableDecl(letOrVarKeyword: .let, bindingsBuilder: {
+                    PatternBinding(pattern: IdentifierPattern(identifier: .identifier("myOtherVar")),
+                                   typeAnnotation: TypeAnnotation(type: SimpleTypeIdentifier("String")))
+                })
+            )
+
+            StructDecl(identifier: "InnerStruct", members: MemberDeclBlock())
+        })
+        )
+
+        let syntax = myStruct.buildSyntax(format: Format())
+        XCTAssertEqual(syntax.description, """
+        struct MyStruct{
+            var myFirstVar: Int
+            let mySecondLet: CGFloat
+            let myOtherVar: String
+            struct InnerStruct{
+            }
+        }
+        """)
+    }
+
+    func testExpressibleAsCodeBlockItem() {
+        let myCodeBlock = SourceFile(eofToken: SyntaxFactory.makeToken(.eof, presence: .present)) {
+            StructDecl(identifier: "MyStruct1", members: MemberDeclBlock())
+
+            StructDecl(identifier: "MyStruct2", members: MemberDeclBlock())
+        }
+
+        let syntax = myCodeBlock.buildSyntax(format: Format())
+        XCTAssertEqual(syntax.description, """
+
+        struct MyStruct1{
+        }
+        struct MyStruct2{
+        }
+        """)
+    }
+
+    func testExpressibleAsSwitchStmt() {
+        let versions = [("version_1", "1.0.0"), ("version_2", "2.0.0"), ("version_3", "3.0.0"), ("version_3_1", "3.1.0")]
+        let expression = IdentifierExpr(identifier: SyntaxFactory.makeIdentifier("version"))
+
+        let switchStmt = SwitchStmt(labelName: nil,
+                                    expression: expression,
+                                    leftBrace: .leftBrace.withTrailingTrivia(.newlines(1)),
+                                    rightBrace: .rightBrace.withLeadingTrivia(.newlines(1)),
+                                    casesBuilder: {
+            for (version, semVer) in versions {
+                SwitchCase(label: SwitchCaseLabel(caseItemsBuilder: {
+                    CaseItem(pattern: EnumCasePattern(caseName: version))
+                }), statementsBuilder: {
+                    ReturnStmt(expression: StringLiteralExpr(semVer, closeQuote: .stringQuote.withTrailingTrivia(.newlines(1))))
+                })
+            }
+        })
+
+        let syntax = switchStmt.buildSyntax(format: Format())
+
+        XCTAssertEqual(syntax.description, """
```

Comment:
I have added this test case so we also have something more than `struct` examples.

BUT is keeps failing, and I don't know why. 
The expected result and the actual result seems equal.
Maybe you can help me @ahoppen ?

<img width="343" alt="Screenshot 2021-09-23 at 22 11 10" src="https://user-images.githubusercontent.com/5477711/134576942-92fd65e3-da6b-40af-854d-dea2e242f771.png">
?

Reply:
The generated code contains trailing whiteface after the `:`. It’s a bit annoying that you can’t have trailing whitespace in the string literal you’re matching against 😬 Maybe trim all trailing whitespace from the generated string before comparing or replace all whitespace with `␣`?

Reply:
Replaced with `␣` it makes it more clean to read the test case I think. 😬